docs/how-to-guides/streams/manage-streams-lab.md

```diff
@@ -0,0 +1,148 @@
+7. Select a transformation procedure and **Attach to Stream**.
+
+You can also develop transformation modules in Python beforehand, in the section
+**Query Modules**. Click on the **New Module**, and the Lab will automatically
+recognize transformation procedures once you define them.
+
+If you developed a procedure in C, you have to [load it into
+Memgraph](manage-streams.md#how-to-create-and-load-a-transformation-module-into-memgraph)
+first, and then you will be able to see it in the **Query Modules** section and
+attach it to a stream. 
+
+Check the transformation module for MovieLens on [Awesome Data
+Stream](https://awesomedata.stream/#/movielens).
+
+<img src={require('../../data/tutorials/create-stream-lab/transformation-module.png').default}className={"imgBorder"}/>
+
+## How to set Kafka configuration parameters?
+
+If necessary, add the Kafka configuration parameters to customize the stream further:
+
+1. In the Kafka Configuration Parameters **+ Add parameter field**.
+2. Insert the parameter name and value.
+3. To add another parameter, **Add parameter filed**.
+4. **Save Configuration** once you have set all parameters.
+
+To connect to the [Awesome Data Stream](https://awesomedata.stream/) you need to set
+the following Kafka configuration parameters:
+
+* **sasl.username** \| public <br/>
+* **sasl.password** \| public <br/>
+* **security.protocol** \| SASL_PLAINTEXT <br/>
+* **sasl.mechanism** \| PLAIN <br/>
+
+<img src={require('../../data/tutorials/create-stream-lab/config-parameters.png').default}className={"imgBorder"}/>
+
+## How to connect Memgraph to the stream and start ingesting the data?
+
+Once the stream is configured, you can **Connect to Stream**. 
+
+Memgraph will do a series of checks, ensuring that defined topics and
+transformation procedures are correctly configured. If all checks pass
+successfully, you can **Start the stream**. Once you start the stream, you will
+no longer be able to change any of the configuration settings, just the
+transformation module. 
+
+The stream status changes to **Running**, and data is ingested into Memgraph.
+You can see the number of nodes and relationships rising as the data keeps
+coming in. If your nodes and relationships numbers stay at zero, check the
+transformation module, as there might be a flaw in the logic that needs to be
+resolved. 
+
+Switch to **Query Execution** and run a query to visualize the data coming in: 
+
+```
+MATCH p=(n)-[r]-(m)
+RETURN p LIMIT 100;
+```
+<img src={require('../../data/tutorials/create-stream-lab/graph.png').default}lassName={"imgBorder"}/>
+
+## How to manage a stream?
+
+To manage a stream in Memgraph Lab, go to **Streams** and click on the stream
+you want to manage. 
+
+### How to start, stop or delete a stream?
+
+To start a draft steam, click on **Connect to Stream**.
+
+To stop or start a stream, click on **Stop Stream**/**Start Stream**.
+
+To delete a stream, click on **Delete Stream**.

```

Comment:
I would transform this in stream view you can stop, start and delete the stream, or something like that. As soon as you say click, it feels strange 😂

Reply:
Sorry! "to click" is official tech writer lingo approved by the holy Google Guide itself... :D I need a verb here, and i need this section for the draft vs. non-draft versions of the stream!